xbmc/GUIInfoManager.h

```diff
@@ -53,6 +53,655 @@ namespace INFO
   class InfoSingle;
 }
 
+// conditions for window retrieval
```

Comment:
@Paxxi @wisler this should have not been merged in again as it was removed already in https://github.com/xbmc/xbmc/commit/9e9396b073ba973b2847896a11244a9f37ecbaff